example/screenshot.dart

```diff
@@ -76,10 +88,10 @@ void captureImage(int hwnd) {
 
     final lpBitmap = calloc<Uint8>(dwBmpSize);
 
-    GetDIBits(hdcWindow, hbmScreen, 0, bmpScreen.ref.bmHeight, lpBitmap,
+    GetDIBits(hdcScreen, hbmScreen, 0, bmpScreen.ref.bmHeight, lpBitmap,
         bitmapInfoHeader.cast(), DIB_RGB_COLORS);
 
-    final hFile = CreateFile(TEXT('captureqwsz.bmp'), GENERIC_WRITE, 0, nullptr,
+    final hFile = CreateFile(TEXT(fileName), GENERIC_WRITE, 0, nullptr,
```

Comment:
If you use an `Arena`, replace this with `fileName.toNativeUtf16(allocator: arena)` or similar.